app/models/tag.rb

```diff
@@ -1,4 +1,8 @@
 class Tag < ActsAsTaggableOn::Tag
+  include ActiveModel::Validations
+
+  validates :name, presence: true
+
```

Comment:
The Tag model inherits from ActAsTagabbleOn::Tag [1] class, which inherits from ActiveRecord. Therefore, there is no need to include the ActiveModel::Validations. The presence validation of the name attribute is defined the same in the ActAsTagabbleOn::Tag class so that we can remove it [and the related test](https://github.com/consul/consul/pull/4837/files#diff-933427331ee759197c461d6677d7dd021f4505c2177f1bd70b60c950e5aa2f8fR34).

[1] https://github.com/mbleigh/acts-as-taggable-on/blob/v8.1.0/lib/acts_as_taggable_on/tag.rb#L3

Reply:
I did not know that. Very good to know